pylabnet/launchers/servers/__init__.py

```diff
@@ -0,0 +1,5 @@
+from os.path import dirname, basename, isfile, join
+import glob
+modules = glob.glob(join(dirname(__file__), "*.py"))
+__all__ = [basename(f)[:-3] for f in modules if isfile(f) and not f.endswith('__init__.py')]
+from . import *
```

Comment:
@mkbhaskar , what is the purpose of this module? Import all modules in this subdirectory? In my case, the `from . import *` proves problematic: Since I don't have the `TimeTagger` package installed, I'll get the following error:  `ModuleNotFoundError: No module named 'TimeTagger'`.

Should we assume that all machines have all the modules in this subdirectory installed? If yes, `TimeTagger` would need to be added to the `setup.yml`.

Reply:
Ok, removed the content from this `__init__`.